python/ray/actor.py

```diff
@@ -235,12 +235,17 @@ def _bind(
         num_returns=None,
         concurrency_group=None,
         _generator_backpressure_num_objects=None,
-    ):
+    ) -> Union[
+        "ray.dag.ClassMethodNode",
+        "ray.dag.TaskReturnNode",
```

Comment:
Ray's normal API returns either one ObjectRef if num_returns == 1 or a tuple of ObjectRefs if num_returns > 1, so for consistency, it's better to always return ClassMethodNode here too.

Reply:
Ack.

Reply:
Done.